docs/installation/octopus-in-container/troubleshooting-octopus-server-in-a-container.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Troubleshooting Octopus Server in a Container
+description: An Octopus Deploy Server instance can be run directly from within a container.
+position: 10
+---
+
+## Accept the EULA
+
+When you create an Octopus Server container, you must agree with the [Octopus Deploy EULA](https://octopus.com/company/legal).
+
+If you get an error similar to:
+
+```
+ERROR: You must accept the EULA
+```
+
+You must pass `--env "ACCEPT_EULA=Y"` when using `docker run` with an Octopus Server image. By passing this value, you accept the Octopus Deploy EULA.
+
+## Windows Containers
+
+When running Windows Containers, you must have Docker set to use Windows Containers and not Linux Containers. Please refer to [switching to Windows Containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers) documentation for switching to Windows Containers.
```

Comment:
This is a bit of a challenging one... Consider rewording a bit to include something about running a windows container on a windows host (as its possible to run both windows and linux containers on windows (as i'm sure you know)).

"if you are running a windows container on a windows host, make sure you have..."

Also, can you include the error message you get when this goes wrong? will help people when googling.